bin/gollum-migrate-tags

```diff
@@ -6,6 +6,24 @@ require 'rubygems'
 
 REPO = ARGV[0] || Dir.pwd
 
+def set_const(const, val)
+  already_defined = begin
+    Object.const_get(const)
+    true
```

Comment:
Need help understanding this `true` here.

Reply:
if `const` is set to `false`, this block would return `false`. But we want to know if the constant has already been defined or not -- not whether the constant evaluates as `true` or `false`.

Motivation: I'm writing a substitute for `defined?` here, as `const` has to be a `String` or `Symbol` here, and `defined?(:some_symbol)` is always true. I wanted a way to not constantly have to do this in the main code: `CONST = bla unless defined?(CONST)`. But on reflection, maybe that's less ugly than this.

The real problem is the use of constant to set application options, but apart from global's it's the only thing I could think of that would allow us to access the options from within the subclassed Filter classes.